tsconfig.json

```diff
@@ -16,7 +16,7 @@
     // "declarationMap": true,                      /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                           /* Generates corresponding '.map' file. */
     // "outFile": "./",                             /* Concatenate and emit output to single file. */
-    "outDir": "./@lib",                             /* Redirect output structure to the directory. */
+    "outDir": "./@scripts",                             /* Redirect output structure to the directory. */
```

Comment:
any specific reason to move this from `@lib` to `@scripts` ? I think @lib makes more sense (since it will also contain, non script files).

Reply:
Yeah, it was mostly to avoid confusion with the `lib` folder on the project.

Reply:
That does make sense. I'm open for your suggestion 🙂

Reply:
I'm kinda liking scripts, I linked a Quest Liason as well for an opinion

Reply:
if it's meant for 3rd party libs, I would probably go with `@scripts` since I tend to use lib as current lib. If it's more than just the 3rd party libs, then I'd go with whatever you decide, I don't have any strong objection on either or